exporter/signalfxexporter/internal/translation/translator.go

```diff
@@ -779,12 +780,7 @@ func filterDimensions(dimensions []*sfxpb.Dimension, withoutDimensions []string)
 
 // dimensionIn checks if the dimension found in the dimensionsKeysFilter
 func dimensionIn(dimension *sfxpb.Dimension, dimensionsKeysFilter []string) bool {
```

Comment:
Could we just remove this method and replace all references with `slices.Contains`?